butterfly-slack/src/main/resources/postAbort.json

```diff
@@ -0,0 +1,23 @@
+{
+  "channel": "%s",
+  "text": "Your Butterfly transformation failed",
+  "as_user": true,
+  "attachments": [
+    {
+      "text": "",
+      "color": "#cb42f4",
+      "actions": [
+        {
+          "text": "Transformation Details",
+          "type": "button",
+          "url": "%s"
+        },
+        {
+          "text": "Contact Us",
+          "type": "button",
+          "url": "http://butterfly.pp-devcos-butterfly.us-central1.gcp.dev.paypalinc.com/contact"
```

Comment:
This also has to be parameterized because it is Pupa specific. Let's add a placeholder here as well, and then use a property for this. Let's call this property `pupa.contactLink`.